pkg/accounting/metrics.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package accounting
+
+import (
+	m "github.com/ethersphere/bee/pkg/metrics"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type metrics struct {
+	// all metrics fields must be exported
+	// to be able to return them by Metrics()
+	// using reflection
+	DebitCount        prometheus.Counter
+	CreditCount       prometheus.Counter
+	DebitEvents       prometheus.Counter
+	CreditEvents      prometheus.Counter
+	CreditDisconnects prometheus.Counter
+	CreditBlocks      prometheus.Counter
+}
+
+func newMetrics() metrics {
+	subsystem := "accounting"
+
+	return metrics{
+		DebitCount: prometheus.NewCounter(prometheus.CounterOpts{
```

Comment:
The naming seems inconsistent. The field is called is called Debit**Count**  but the metric is total_debited_**amount**.  Further down there is actually a total_debit_events_**count** metric.

`DebitEvent` should probably be `DebitCount` and `DebitCount` `DebitAmount`

Reply:
@ralph-pichler Agreed, we need a better naming scheme.

Reply:
I would propose to rename the counters/metrics in the following way:

- `TotalDebitedAmount` to track `total_debited_amount`
- `TotalCreditedAmount` to track `total_credited_amount`
- `DebitEventsCount` to track `debit_events_count`
- `CreditEventsCount` to track `credit_events_count`
- `AccountingDisconnectsCount` to track `accounting_disconnects_count`
- `AccointingBlocksCount` to track `accounting_blocks_count`

Reply:
Sounds good to me